### DISCUSSION TITLE
Appropriate segmentation metric + loss

### DISCUSSION THREAD

**Discussion by vikashg** [Project-MONAI/MONAI#7119] [Q&A]
Hello all, 
I would like some feedback on the following segmentation problems. What you are seeing here are air trappings in the lungs. 
I used models like SegResNet and DenseNet with metrics like DiceScore and GeneralizedDiceScore. I used DiceLoss and binary cross entropy as the loss function. But the results were not good. The Dice doesn't move beyond 0.6. 

My understanding is that because the segmentation is not a contiguous blob but more akin to an archipelago, the dice score is getting confused and probably there is no appropriate measure for such a segmentation. 
Here is the ground truth
![image](https://github.com/Project-MONAI/MONAI/assets/3863212/044d9338-8cdb-4cf7-a556-53fecc7e065a)
Here is a segmentation result.
![image](https://github.com/Project-MONAI/MONAI/assets/3863212/c7dc89c7-c584-4d4e-a920-e5b92c040e89)

**Comment by vikashg** [comment]
Any thoughts/ideas will be greatly appreciated.
